### PR TITLE
DOCAPI-7400: Gorilla codecs. Basic review of English text.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5600.

### CODE REVIEW

File: docs/en/query_language/create.md [hunk 1]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
 
 Supported compression algorithms:
 
-- `NONE` - no compression for data applied
+- `NONE` — no compression for data applied.
```
Comment (blinkov) [inline docs/en/query_language/create.md]: **is** applied

File: docs/en/query_language/create.md [hunk 2]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
...
 - `LZ4`
```
Comment (blinkov) [inline docs/en/query_language/create.md]: brief description, link to full description, mention that it's the default

File: docs/en/query_language/create.md [hunk 3]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
...
-- `LZ4HC(level)` - (level) - LZ4\_HC compression algorithm with defined level.
+- `LZ4HC(level)` — (level) - LZ4\_HC compression algorithm with defined level.
```
Comment (blinkov) [inline docs/en/query_language/create.md]: link to algorithm description

File: docs/en/query_language/create.md [hunk 4]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
...
 Possible `level` range: \[3, 12\]. Default value: 9. Greater values stands for better compression and higher CPU usage. Recommended value range: [4,9].
```
Comment (blinkov) [inline docs/en/query_language/create.md]: two ranges are written inconsistently: one has backslashes and space, the other do not

File: docs/en/query_language/create.md [hunk 5]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
...
-- `ZSTD(level)` - ZSTD compression algorithm with defined `level`. Possible `level` value range: \[1, 22\]. Default value: 1.
+- `ZSTD(level)` — ZSTD compression algorithm with defined `level`. Possible `level` value range: \[1, 22\]. Default value: 1.
```
Comment (blinkov) [inline docs/en/query_language/create.md]: link

File: docs/en/query_language/create.md [hunk 6]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
...
 Greater values stands for better compression and higher CPU usage.
-- `Delta(delta_bytes)` - compression approach when raw values are replace with difference of two neighbour values. Up to `delta_bytes` are used for storing delta value.
+- `Delta(delta_bytes)` — compression approach when raw values are replace with difference of two neighbour values. Up to `delta_bytes` are used for storing delta value.
```
Comment (blinkov) [inline docs/en/query_language/create.md]: as far as I understand delta_bytes is not only the maximum size of delta, but also a byte size of raw values to subtract (which is important to understand to use this properly)

File: docs/en/query_language/create.md [hunk 7]
```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
...
+- `Gorilla` — Compresses values down to 1 bit (in the best case). The codec is efficient when storing series of floating point values that change slowly, because the best compression rate is achieved when neighbouring values are binary equal. Implements the algorithm used in Gorilla TSDB, extending it to support 64 bit types. For additional information, see the "Compressing values" section of the [Gorilla: A Fast, Scalable, In-Memory Time Series Database](http://www.vldb.org/pvldb/vol8/p1816-teller.pdf) document.
 
 Syntax example:
+
 ```
 CREATE TABLE codec_example
 (
-    dt Date CODEC(ZSTD), /* используется уровень сжатия по-умолчанию */
+    dt Date CODEC(ZSTD), /* Uses the default compression level. */
```
Comment (blinkov) [inline docs/en/query_language/create.md]: why this comment is here instead of ZSTD description above?

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/query_language/create.md (hunk 3 above)
Comment (BayoNet) [reply docs/en/query_language/create.md]: In looks that there is no algorithm https://github.com/lz4/lz4/issues/660. Do we support it still?

File: docs/en/query_language/create.md [hunk 8]
```diff
@@ -107,22 +107,22 @@ Besides default data compression, defined in [server settings](../operations/ser
...
-- `Gorilla` - stores (parts of) xored values in compact binary form, compressing values down to 1 bit (in the best case). Best compression rate is achieved when neighbouring values are binary equal. Basic use case - floating point data that do not change rapidly. Implementation is based on [Gorilla paper](http://www.vldb.org/pvldb/vol8/p1816-teller.pdf), and extended to support 64bit types.
+- `Delta(delta_bytes)` — compression approach, when raw values are replaced with the difference of two neighbour values. Up to `delta_bytes` are used for storing delta value, so `delta_bytes` is a maximum size of raw values.
+Possible `delta_bytes` values: 1, 2, 4, 8. Default value for `delta_bytes` is `sizeof(type)`, if it is equals to 1, 2, 4, 8. Otherwise it equals 1.
```
Comment (blinkov) [inline docs/en/query_language/create.md]: is equals to -> is equal to

**Pull request merged**